imitation/util/trainer.py

```diff
@@ -17,15 +17,15 @@
 @gin.configurable
 def init_trainer(env_id, policy_dir, use_gail, use_random_expert=True,
                  **kwargs):
-  """
-  Build an AIRLTrainer, ready to be trained on a vectorized environment
+  """Build an AIRLTrainer, ready to be trained on a vectorized environment
   and either expert rollout data or random rollout data.
 
-  env_id (str): The string id of a gym environment.
-  use_random_expert (bool):
-    If True, then use a blank (random) policy to generate rollouts.
-    If False, then load an expert policy. Will crash if DNE.
-  **kwargs -- Pass additional arguments to the AIRLTrainer constructor.
+  Args:
+    env_id (str): The string id of a gym environment.
+    use_random_expert (bool):
+        If True, then use a blank (random) policy to generate rollouts.
+        If False, then load an expert policy. Will crash if DNE.
```

Comment:
Needs docstring for policy_dir and use_gail (specifically clarifying what policy_dir is meant to be if use_random_expert=True, since it seems like in that case we wouldn't be loading a policy?)

Reply:
Yeah, kinda confusing that AIRLTrainer can `use_gail` here, now that I think about it. Nevan and I weren't very careful about cleanliness during the review for this change/